cli/tools/lint/mod.rs

```diff
@@ -525,7 +525,16 @@ fn handle_lint_result(
   let mut reporter = reporter_lock.lock();
 
   match result {
-    Ok((_source, mut file_diagnostics)) => {
+    Ok((source, mut file_diagnostics)) => {
+      if !source.diagnostics().is_empty() {
+        for parse_diagnostic in source.diagnostics() {
+          log::info!("{}: {}", colors::red("error"), parse_diagnostic);
```

Comment:
Maybe this should be `colors::yellow("warn")` since it's not a fatal error?

Also, perhaps we should just remove the "found x parsing related problems" message?

Reply:
Surely, suggested changes are done and pushed.
I'd also like to point out that I previously aimed to follow the same approach used for reporting/logging Lint diagnostic errors (which logs error in red color and prints the number of Lint diagnostic errors).